searx/searx
# Onesearch engine

## What does this PR do?

Add a new search engine https://www.onesearch.com/

## Why is this change important?
Add another search engine to have a more resilient pool of engines in case of failures

## How to test this PR locally?

1. Run `sudo make run`
1. Go to `/preferences`
1. Disable all search engines with the exception of onesearch (the last one)
1. Make some search
1. Assert that it works as expected
1. Assert that the pagination works

## Author's checklist

This search engine is ON by default, since they claim to be privacy friendly

## Related issues

[/e/ foundation spot - issue 3676](https://gitlab.e.foundation/e/backlog/-/issues/3676)
